### DISCUSSION TITLE
Biometric method selection

### DISCUSSION THREAD

**Discussion by aloniado** [NiklasMerz/cordova-plugin-fingerprint-aio#300] [Ideas]
### Feature Request
Biometric method selection
#### Feature Description
Hi,
I think a way to select a biometric identification method will be useful.
I would like to use only fingerprints, and can't find a way to disable face recognition.

Thank you

**Comment by NiklasMerz** [comment]
We are somewhat limited to what the Biometric API offer on each OS. Please comment here with details of features available on Android and iOS.
